terraform/deployments/cluster-infrastructure/outputs.tf

```
@@ -121,3 +121,7 @@ output "monitoring_namespace" {
 output "clamav_db_efs_id" {
   value = aws_efs_file_system.clamav-db.id
 }
+
+output "public_ips_nat_gateways" {
```

optional: `public_nat_gateway_ips` might be slightly clearer name
```suggestion
output "public_nat_gateway_ips" {
```